tests/test_F1_plotting.py

```diff
@@ -0,0 +1,210 @@
+    sep=";",
+    header=0,
+    index_col=0,
+)
+
+# data for test_store_barchart_for_capacities
+DICT_KPI = {
+    "kpi": {
+        "scalar_matrix": pd.DataFrame(
+            {"label": ["asset_a", "asset_b"], "optimizedAddCap": [1, 2]}
+        )
+    },
+}
+
+
+class TestNetworkx:
+    def test_if_networkx_graph_is_stored_save_plot_true(self):
+        @mock.patch(
+            "argparse.ArgumentParser.parse_args",
+            return_value=PARSER.parse_args(
+                ["-i", TEST_INPUT_PATH_NX_TRUE, "-o", TEST_OUTPUT_PATH, "-ext", "csv"]
+            ),
+        )
+        def setup_module(m_args):
+            """Run the simulation up to module E0 and save dict_values before and after evaluation"""
+            if os.path.exists(TEST_OUTPUT_PATH):
+                shutil.rmtree(TEST_OUTPUT_PATH, ignore_errors=True)
+            user_input = initializing.process_user_arguments()
+
+            logging.debug("Accessing script: B0_data_input_json")
+            dict_values = data_input.load_json(
+                user_input["path_input_file"],
+                path_input_folder=user_input["path_input_folder"],
+                path_output_folder=user_input["path_output_folder"],
+                move_copy=False,
+            )
+            logging.debug("Accessing script: C0_data_processing")
+            data_processing.all(dict_values)
+
+            logging.debug("Accessing script: D0_modelling_and_optimization")
+            results_meta, results_main = modelling.run_oemof(dict_values)
+
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "network_graph.png")) is True
+
+    def test_if_networkx_graph_is_stored_save_plot_false(self):
+        @mock.patch(
+            "argparse.ArgumentParser.parse_args",
+            return_value=PARSER.parse_args(
+                ["-i", TEST_INPUT_PATH_NX_FALSE, "-o", TEST_OUTPUT_PATH, "-ext", "csv"]
+            ),
+        )
+        def setup_module(m_args):
+            """Run the simulation up to module E0 and save dict_values before and after evaluation"""
+            if os.path.exists(TEST_OUTPUT_PATH):
+                shutil.rmtree(TEST_OUTPUT_PATH, ignore_errors=True)
+            user_input = initializing.process_user_arguments()
+
+            logging.debug("Accessing script: B0_data_input_json")
+            dict_values = data_input.load_json(
+                user_input["path_input_file"],
+                path_input_folder=user_input["path_input_folder"],
+                path_output_folder=user_input["path_output_folder"],
+                move_copy=False,
+            )
+            logging.debug("Accessing script: C0_data_processing")
+            data_processing.all(dict_values)
+
+            logging.debug("Accessing script: D0_modelling_and_optimization")
+            results_meta, results_main = modelling.run_oemof(dict_values)
+
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "network_graph.png")) is False

```

Comment:
```suggestion
        assert os.path.exists(os.path.join(TEST_OUTPUT_PATH, "network_graph.png")) is False

```

might explain the failing test :)

Reply:
Ouch.